server/http/helloapp/command.go

```diff
@@ -0,0 +1,153 @@
+package helloapp
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/pkg/errors"
+)
+
+type SubCommand struct {
+	AppID       string        `json:"app_id"`
+	Pretext     string        `json:"pretext"`
+	Description string        `json:"description"`
+	SubCommands []SubCommand  `json:"sub_commands"`
+	Args        []interface{} `json:"args"`
+	FormURL     string        `json:"form_url"`
+}
+
+func (h *helloapp) handleHelloCommandSubmission(w http.ResponseWriter, req *http.Request, claims *apps.JWTClaims, call *apps.Call) (int, error) {
+	var e = func(err error, msg string) (int, error) {
```

Comment:
Wonder how useful is this function, so we have it outside of this function and named. If we are going to create more commands handlers may be interesting (even if we are just using them in the sample helloapp).

Reply:
Yes this was for quick debugging, and should be removed from this function